sw/device/silicon_creator/lib/drivers/pinmux.c

```diff
@@ -126,10 +126,10 @@ static void configure_output(pinmux_output_t output) {
 }
 
 void pinmux_init(void) {
-  uint32_t bootstrap_en =
-      otp_read32(OTP_CTRL_PARAM_OWNER_SW_CFG_ROM_BOOTSTRAP_EN_OFFSET);
-  if (launder32(bootstrap_en) == kHardenedBoolTrue) {
-    HARDENED_CHECK_EQ(bootstrap_en, kHardenedBoolTrue);
+  uint32_t bootstrap_dis =
+      otp_read32(OTP_CTRL_PARAM_OWNER_SW_CFG_ROM_BOOTSTRAP_DIS_OFFSET);
+  if (launder32(bootstrap_dis) != kHardenedBoolTrue) {
```

Comment:
It strikes me that although the logic is equivalent, the change from == to != might be relevant to hardening.

I think this may actually be a positive change because it's now harder to skip the branch  by bitflipping `bootstrap_dis`.

Reply:
I think it it depends on what you consider an asset. In this case, being able to bootstrap is the asset, so this change actually makes it easier to be able to bootstrap the chip (2**32-1 values vs just 1 value to disable it) but this was a conscious decision to maximize our chances of bootstrapping.